woocommerce/woocommerce-gateway-stripe
# refactor: keep track of UPE disabled

# Changes proposed in this Pull Request:

In WCPay we keep track of whether UPE has been manually disabled so that we don't show the inbox notice anymore.

This does the same - I forgot to add it to the requirements in the inbox ticket.

Now, when UPE is disabled through the UI, the inbox notice is no longer displayed again.

Similar approach to https://github.com/Automattic/woocommerce-payments/pull/2568

# Testing instructions
- Ensure you have UPE disabled
- Ensure you have `_wcstripe_feature_upe_settings` enabled
- Go to the WC home: http://localhost:8082/wp-admin/admin.php?page=wc-admin
- On the inbox note, click "Enable in your store"
- On the wizard, enable UPE
- On the Stripe settings, disable UPE via the hamburger menu
- Now the inbox notice should no longer be displayed again
- UPE can still be enabled through the onboarding wizard
